internal/server/place/golden/get_related_locations/state.json

```diff
@@ -0,0 +1,8 @@
+{
+  "data": {
+    "CumulativeCount_MedicalConditionIncident_COVID_19_ConfirmedOrProbableCase": {
```

Comment:
Wonder why this changes?  Is it because this merge happens for current branch too?

Reply:
This is a new test added to cover merge case for dc_branch table